### PR TITLE
RK-19777 - remove sentry

### PR DESCRIPTION
**Pull request by ElDuderinos**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -6,19 +6,12 @@
...
 from todos_store import Store
-import sentry_sdk
-from sentry_sdk.integrations.flask import FlaskIntegration
 from jaeger_client import Config
 from flask_opentracing import FlaskTracer
 from utils.logging import on_add_todo_logging, on_get_todos_logging
 import os 
 from flask import send_from_directory     
 
-sentry_sdk.init(
```
Comment (Urook) [inline app.py:17]: Dont we need to recalculate the hash of the file?

Comment (ElDuderinos) [reply app.py:17]: true true, fixed

File: app.py [hunk 2]
```diff
@@ -6,19 +6,12 @@
 from datetime import datetime
 from random import randint
 from todos_store import Store
-import sentry_sdk
-from sentry_sdk.integrations.flask import FlaskIntegration
```
Comment (Urook) [inline app.py:10]: Oh no,
Just realized our e2es rely on things being in specific lines. So, removing lines will break e2es. Lets leave the removed lines empty insteqd of removing them?

**Review by Urook — CHANGES_REQUESTED** [review]

File: app.py (hunk 2 above)
Comment (ElDuderinos) [reply app.py:10]: @Urook ?

Comment (Urook) [reply app.py:10]: @ElDuderinos Yes, in profiling

Comment (ElDuderinos) [reply app.py:10]: Gotcha, done

File: app.py [hunk 3]
```diff
@@ -7,17 +7,17 @@
 from random import randint
 from todos_store import Store
 import sentry_sdk
```
Comment (Urook) [inline app.py:9]: Why not remove this?

**Review by Urook — CHANGES_REQUESTED** [review]

File: app.py (hunk 3 above)
Comment (ElDuderinos) [reply app.py]: aha, sorry good catch, removed

**Review by Urook — APPROVED** [review]

**Pull request merged**